.github/workflows/system-tests.yml

```diff
@@ -30,88 +22,120 @@ jobs:
       WEBLOG_VARIANT: ${{ matrix.weblog-variant }}
       DD_API_KEY: ${{ secrets.DD_API_KEY }}
     steps:
+      - name: Filter paths
+        uses: dorny/paths-filter@v2
```

Comment:
perhaps we can use the same approach taken for the framework tests to avoid using this unapproved action?